packages/amplication-client/src/Project/ArchitectureConsole/hooks/useModelOrganizer.tsx

```diff
@@ -505,6 +507,10 @@ const useModelOrganization = ({ projectId, onMessage }: Props) => {
     [setNodes, setEdges, edges, nodes]
   );
 
+  const handleDuplicateNameDialogClicked = useCallback(() => {
```

Comment:
not sure why we see UI events in the hook... I didn't expect to see dialog click in the hook

Reply:
use the name  clearDuplicateEntityError and the function should always set the value to false